src/client/app/utils/getTemperatureUnitConversion.ts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export enum TemperatureUnitType {
+	celsius = 'celsius',
+	fahrenheit = 'fahrenheit'
+}
+
+/**
+ * Gets the conversion between two temperature units
+ * @param fromUnit unit to convert FROM
+ * @param toUnit unit to convert TO
+ * @returns conversion multiplier, or zero if conversion to none
+ */
+export function getTemperatureUnitConversion(fromUnit: TemperatureUnitType, toUnit: TemperatureUnitType): number {
```

Comment:
I could not find where this is used. OED can convert between units so I'm wondering if this needed or if OED should always add a conversion for this if it is needed.